docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -283,3 +283,54 @@ GET /transactions/{tx_id}
 
    :statuscode 200: A transaction with that ID was found.
    :statuscode 404: A transaction with that ID was not found.
+
+
+GET /unspents/
+-------------------------
+
+.. http:get:: /unspents?owner_after={owner_after}
```

Comment:
Since the only parameter here is `owners_after`, why use a query parameter at all.
Why not just do like the transactions endpoint `.. http:get:: /unspents/{owners_after}`